src/components/pages/Vehicles.jsx

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable */
+import React, { useEffect, useState } from 'react'
+import './styles/vehicle.css'
+// import img from '../../assets/bike.jpg'
+import { useDispatch, useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom'
+import { getVehicles } from '../../redux/vehicles/vehicles';
+// Import Swiper React components
+import { Swiper, SwiperSlide } from 'swiper/react';
+// Import Swiper styles
+import 'swiper/css';
+// import { createAsyncThunk } from '@reduxjs/toolkit';
+// import VehiclesDetails from './VehiclesDetails';
+// import { getVehicleId } from '../../redux/vehicles/vehicles';
+import { Navigation, Pagination, Scrollbar, A11y } from 'swiper';
+import 'swiper/css/navigation';
+import 'swiper/css/pagination';
+import 'swiper/css/scrollbar';
+
+const Vehicles = () => {

```

Comment:
- [ ] OPTIONAL: When I want to click on a motorcycle to view its details, I need to click on a little number each time:

![image](https://user-images.githubusercontent.com/57421823/227379649-9e366b21-ffdb-4646-9af9-cf8b85fc7cd7.png)

I suggest you make the link embedded in the image, so the motorcycles are clickable, and the UX will be vastly improved for your users. 💪

Reply:
Removed number and add links to image